lib/bolt/cli.rb

```diff
@@ -392,7 +392,7 @@ def execute(options)
         output_format: config.format,
         # For continuity
         boltdir_type: config.project.type
-      }
+      }.merge!(analytics.plan_counts(config.project.path))
```

Comment:
```suggestion
      }.merge!(analytics.plan_counts(config.project.plans_path))
```

Reply:
I don't remember why I didn't do this initially but I know there was a reason....anyway, maybe it wasn't important